apps/www/_blog/2025-09-18-postgrest-13-release.mdx

```diff
@@ -0,0 +1,141 @@
+
+## Automatic tsvector convertion
+
+Previously you could only use the full text search operator on `tsvector` columns, now you can do it on `text` columns too:
+
+```js
+const { data, error } = await supabase.from('titles').textSearch('primary_name', `'god' & 'father'`)
+```
+
+This works because `text` columns will be automatically converted with `to_tsvector`.
+
+## Max Affected
+
+You can now limit to the amount of rows affected by an `update` or `delete` operation with `maxAffected`:
+
+```js
+const { data, error } = await supabase
+  .from('people')
+  .update({ primary_name: 'Marlon Brando Jr.' })
+  .eq('nconst', 'nm0000008')
+  .maxAffected(1)
+```
+
+If the rows affected by the operation surpass the limit in `maxAffected`, an error will be thrown.
+
+This also works with `rpc()`, given that it modifies rows and returns the affected rows. More on details on the [official docs](https://docs.postgrest.org/en/v13/references/api/preferences.html#max-affected).
+
+## Content-Length header
+
+For observability, you can now verify the response body size in bytes in the `Content-Length` header.
+
+```bash
+
+HTTP/1.1 200 OK
+Content-Length: 104
+Content-Location: /items
+```
+
+This helps in cases where you want to know which requests consume the most traffic to avoid exceeding egress limits.
+
+## Proxy-Status header
+
+The PostgREST error code is now present in the `Proxy-Status` header.
+
+```bash
+HTTP/1.1 406 Not Acceptable
+Proxy-Status: PostgREST; error=PGRST116
+```
+
+You can check this header and the Content-Length one the Supabase Logs Explorer.
+
+## Breaking Changes
+
+### Limited Update/Delete
+
+Previously on v12, you could limit the updated or deleted affected rows with a combination of the `limit()` and `order()` methods.
+
+```js
+const { data, error } = await supabase
+  .from('people')
+  .update({ primary_name: 'Marlon Brando Jr.' })
+  .eq('primary_name', 'Marlon Brando')
+  .limit(1)
+  .order('nconst')
+```
+
+But this never gave an error, it only did the equivalent of a `UPDATE .. LIMIT` SQL statement. It also required adding a PK in the `order` method, which was a confusing interface.
+
+To do this now, you should use the new Max Affected instead.
+
+### JWT kid validation
+
+PostgREST now validates the JWT `kid` claim. If your JWT contains a `kid`, it will try to match the `kid` of one of the keys in the configured JSON Web Key Set. Check the [official docs](https://docs.postgrest.org/en/v13/references/auth.html#jwk-kid-validation) for more details.
+
+If you use Supabase Auth or the CLI to create JSON Web Keys, you shouldn’t worry about this change. For users that integrate with other Auth systems, check that both your JWT and JWKS follow the above rules.
+
+### Schema validation in PostgREST search path
+
+The schemas inside `db-schemas` and `db-extra-search-path` are now validated. This means you cannot put a nonexistent schema there, if you do PostgREST will faill with an error message.
```

Comment:
```suggestion
The schemas inside `db-schemas` and `db-extra-search-path` are now validated. This means you cannot put a nonexistent schema there, if you do PostgREST will fail with an error message.
```

questions:

- what happens if people upgrade and the schema doesnt exist? will the upgrade fail?
- what happens if someone drops a schema after it's been added to PostgREST?